dialplan/asterisk/extensions_lib_features.conf

```
@@ -194,10 +194,12 @@ same  =   n,Set(CHANNEL(language)=${IMPORT(${TRANSFERERNAME},CHANNEL(language))}
 same  =   n(end),Return()
 
 exten = a,1,Wait(1)
+same  =   n,GotoIf($["${XIVO_VM_PASSWORD}" = ""]?start-vm)
 same  =   n,GotoIf($["${XIVO_MAILBOX_LANGUAGE}" = ""]?$[${PRIORITY} + 2])
 same  =   n,Set(CHANNEL(language)=${XIVO_MAILBOX_LANGUAGE})
 same  =   n,VoiceMailMain(${XIVO_MAILBOX}@${XIVO_MAILBOX_CONTEXT})
 same  =   n,Hangup()
+same  =   n(start-vm),Goto(voicemail-app)
```

Did you test with label?
When you when to go outside your current `exten`, the first argument of `Goto` should be the exten.
```suggestion
same  =   n(start-vm),Goto(voicemail,voicemail-app)
```